ipapython/install/common.py

```diff
@@ -89,13 +89,24 @@ class Interactive(core.Configurable):
 
 
 class Continuous(core.Configurable):
+    def __init__(self, **kwargs):
+        super(Continuous, self).__init__(**kwargs)
+        self.exceptions_handled = 0
+
     def _handle_execute_exception(self, exc_info):
         try:
             super(Continuous, self)._handle_execute_exception(exc_info)
+        except SystemExit:
+            pass
```

Comment:
Please don't.

Reply:
I could find no other way because of the way this is architected. sys.exit() is an exception unto itself if not caught will result in a nasty traceback.